Automattic/wp-calypso
# [FSE] Remove alignment option for post content block.

This, along with theme style changes, allows the post content to look like it does in the non-fse editor.

#### Changes proposed in this Pull Request
* Disables alignment options for the post content block.

#### Testing instructions
1. Activate the FSE plugin with this PR.
2. Open a page and select the post content block.
3. There should not be any options to change the alignment of the block.
4. The content should show up in the middle of the page with significant margins. Before, it would have been edge to edge.
5. _Update_: Also test that alignment options are disabled for the header and footer.

See also this theme PR which makes the styles of the post content work properly with alignment disabled: https://github.com/Automattic/themes/pull/1072

#### Followup tasks:
- [x] Disable alignment options for the header/footer
- [x] Make sure the header/footer styles are updated to work with alignment disabled

Fixes #34558
Coincidentally, this will also resolve #34430
